src/Tilemap.ts

```diff
@@ -177,6 +178,7 @@ export class Tilemap extends Container
      *      per row.
      * @param [options.animCountY=1024] - For animated tiles, this is the number of animation frame textures
      *      per column.
+     * @param [options.animDuration=1] - For animated tiles, this is the animation duration of the animated tile
```

Comment:
This doesn't explain how animation-duration behaves. I don't get it either. The `tileAnim` basically sets the animation frame to use; if you specify a duration, how does it change that? i.e. if `tileAnim` is (1, 2) and `animDuration` for a specific tile is 2, then which animation frame is choosen?

Reply:
@blurymind It's true that this would some thinking about the naming and the description of this "duration". A duration is a time, but this parameter seems not a time, rather a "aniamtion speed scale" or a "animation speed factor"? This number seems not to have any unit, hence the "factor" word (but maybe there is a better alternative).
Also maybe you could explain how this is stored by Tiled/Ldtk to see if we're using the same concept as they do for animated tiles? :)

Reply:
Also, `tileAnim` was designed to be 2D (animation frame can be specified in both X and Y direction). The factor should be able work with that (it seems 1D right now).

Reply:
> Also, `tileAnim` was designed to be 2D (animation frame can be specified in both X and Y direction). The factor should be able work with that (it seems 1D right now).

for animation speed I prefer to keep it one dimentional as it makes no sense to play frames at different speeds on X and Y and would just make this complicated and hard to use/maintain imo

Reply:
> @blurymind It's true that this would some thinking about the naming and the description of this "duration". A duration is a time, but this parameter seems not a time, rather a "aniamtion speed scale" or a "animation speed factor"? This number seems not to have any unit, hence the "factor" word (but maybe there is a better alternative).

It is literally the duration a frame stays on the screen before it is replaced by the next frame. You are correct that it acts as a factor in our case, so it would make sense to call it frameDurationFactor? If you use the word "speed" it would imply that a higher value gives higher speed of playback - which is not actually true. A higher value makes the playback slower

Reply:
`animationDivisor`?

Reply:
I am fine with any name really :) @4ian @ivanpopelyshev does `animationDivisor` sound ok?
Its easy for me to change it, just want to be sure we all agree on it

Reply:
I want to get this PR through faster; even though it's called 'frame duration' in Tiled, are they encoding the same value? The `tileAnim` animation vector isn't associated with time at all; it tells which animation frame to use at any point in time.

`animationDivisor` makes sense because you divide the `tileAnim` vector by the "divisor".

Reply:
now renamed to animDivisor and tileAnimDivisor for the method. We are no longer passing it with tileAnimX and tileAnimY